.github/workflows/openconcept.yaml

```diff
@@ -50,11 +47,10 @@ jobs:
         mv n3* ./openconcept/components/empirical_data/n+3
     - name: Build and Test
       run: |
-        python -m pytest --cov-config .coveragerc --cov=openconcept
-    - name: Push to Coveralls
-      if: ${{ matrix.os == 'ubuntu-latest' }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        pip install coveralls
-        coveralls --service=github --rcfile=.coveragerc
+        python -m pytest --cov-config=.coveragerc --cov=openconcept --cov-report=xml
```

Comment:
Somewhat unrelated, but the Windows/Linux download can be combined together since `curl` is available on all the images (I think)